image_width: 752
image_height: 480
camera_name: cam3
camera_matrix:
  rows: 3
  cols: 3
  data: [398.9944782, 0.0, 364.41599021, 0.0, 398.080578743, 275.477777432, 0.0, 0.0, 1.0]
distortion_model: equidistant
distortion_coefficients:
  rows: 1
  cols: 4
  data: [-0.0560625387897, -0.014037695708, 0.00638405850267, -0.00151939229656]

